nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/packages.xml

```diff
@@ -1,12 +1,19 @@
 <packages>
-  <packageDefinitions baseUrl="http://cdn.nuxeo.com/nuxeo-8.10-SNAPSHOT/mp/">
+  <packageDefinitions baseUrl="http://cdn.nuxeo.com/nuxeo-8.4-SNAPSHOT/mp/">
     <!-- Virtual Packages -->
     <package id="nuxeo-ui-virtual" label="User interface" color="#635959" textcolor="#FFF"
       shortlabel="UI" virtual="true" />
     <package id="nuxeo-addon-virtual" label="Additional addons" color="#635959" textcolor="#FFF"
       shortlabel="ADDON" virtual="true" />
     <package id="nuxeo-server" label="Server" color="#635959" textcolor="#FFF"
       shortlabel="SERVER" virtual="true" />
+
+    <!-- virtual package to handle no-ui -->
+    <package id="nuxeo-no-ui" label="No-UI" color="#635959" textcolor="#FFF"
+      shortlabel="NOUI" virtual="true"
+      desc="Install Nuxeo without any UI; you'll still be able to use it as a standalone server; use clients to call the REST API, Automation API, ..."/>
```

Comment:
I guess that we usually prefer writing "you will". @sguitter ?
You should ask for a review of the whole sentence.